ocaml/xapi/xapi_clustering.ml

```diff
@@ -535,10 +535,10 @@ module Watcher = struct
 
   let cluster_change_watcher : bool Atomic.t = Atomic.make false
 
-  (* this is the time it takes for the update request to time out. It is ok to set
+  (* This is the time it takes for the update request to time out. It is ok to set
      it to a relatively long value since the call will return immediately if there
-     is an update *)
-  let cluster_change_interval = Mtime.Span.min
+     is an update. *)
+  let cluster_change_interval = Mtime.Span.(5 * min)
```

Comment:
May I know how the 5 minutes is determined for the updates check timeout, thank you!

Reply:
No, thank you! Just for curiosity~